utilities/src/main/java/io/syndesis/qe/utils/PublicApiUtils.java

```diff
@@ -0,0 +1,48 @@
+package io.syndesis.qe.utils;
+
+import cz.xtf.openshift.OpenShiftBinaryClient;
+import io.fabric8.kubernetes.api.model.ServiceAccount;
+import io.syndesis.qe.TestConfiguration;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.fail;
+
+@Slf4j
+public class PublicApiUtils {
+
+    private static final String SERVICE_ACCOUNT_NAME = "syndesis-cd-client";
+    private static String publicApiToken;
+
+    private PublicApiUtils() {
+    }
+
+    /**
+     * Method creates service account for Public API and gives them competent role
+     */
+    public static void createServiceAccount() {
+        Optional<ServiceAccount> existServiceAccount = OpenShiftUtils.client().serviceAccounts().list().getItems().stream()
+                .filter(t -> t.getMetadata().getName().equalsIgnoreCase(SERVICE_ACCOUNT_NAME))
+                .findFirst();
```

Comment:
when you know the name, you don't need to iterate over the list and you can use `OpenshiftUtils.client().serviceAccounts().withName("").get()`